src/openrct2/management/finance.c

```diff
@@ -94,12 +94,12 @@ void finance_payment(money32 amount, rct_expenditure_type type)
  */
 void finance_pay_wages()
 {
-    rct_peep* peep;
```

Comment:
Unrelated change, and variables are declared on top for a good reason. Please revert.